app/src/main/java/com/example/campusguide/map/displayIndoor/BuildingInfo.kt

```diff
@@ -0,0 +1,72 @@
+package com.example.campusguide.map.displayIndoor
+
+import com.example.campusguide.map.GoogleMapAdapter
+import com.google.android.gms.maps.model.BitmapDescriptorFactory
+import com.google.android.gms.maps.model.GroundOverlay
+import com.google.android.gms.maps.model.GroundOverlayOptions
+import com.google.android.gms.maps.model.LatLng
+
+class BuildingInfo constructor(buildingName: String, map: GoogleMapAdapter) {
+
+    private val floors: IntArray? = setFloors(buildingName)
+        fun getFloors(): IntArray? {
+            return floors
+        }
+    private val buildingImageCoordinates: LatLng = setBuildingImageCoordinates(buildingName)
+    fun getBuildingImageCoordinates(): LatLng {
+        return buildingImageCoordinates
+    }
+    private val floorPlans: HashMap<Int, GroundOverlay>? = setUpFloorPlans(buildingName, map)
+        fun getFloorPlans(): HashMap<Int, GroundOverlay>? {
+            return floorPlans
+        }
```

Comment:
same indentation confusion for me here ?_?

Reply:
^ see above